### DISCUSSION TITLE
Hi Veda

### DISCUSSION THREAD

**Discussion by yuangiving** [VedaOrg/Veda-docs#4] [General]
I am very interesting in the Bitcoin EVM extension. Looking forward to the testing version.

**Comment by Raylin51** [comment]
We are accelerating our development and expect to launch a beta version within a few weeks.

**Comment by forfan33** [comment]
加油，看好你们

**Comment by yedoudada** [comment]
I'm looking forward to it.

**Comment by siweibozhe** [comment]
....

**Comment by liboheng** [comment]
废了，被抹杀了哎
